deployment/CP15MOAS-GL387_Deploy.csv

```diff
@@ -0,0 +1,7 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+AR82,,,,CP15MOAS-GL387-00-ENG000000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-01-ADCPAM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-02-FLORTM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04471,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-03-CTDGVM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09059,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-04-DOSTAM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00310,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-05-PARADM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.88224,-74.83170,,350,350,MA,,
```

Comment:
Looks good.  (I calculated 35.88223 from the QCT, but perhaps that was calculated from the ELog).

Reply:
yes I took it from the elog (the QLR table is calculated from that & rounded) - I threw a screenshot fr elog up in slack data channel.